vars/capsuleSystemTests.groovy

```diff
@@ -0,0 +1,207 @@
+/* groovylint-disable 
+  BuilderMethodWithSideEffects, CompileStatic, DuplicateStringLiteral, 
+  FactoryMethodName, VariableTypeRequired */
+void buildGoBinary(String directory, String outputBinary, String packages) {
+  timeout(time: 5, unit: 'MINUTES') {
```

Comment:
I wouldn't set build timeouts, builds breaks but do not take forever

Reply:
its hanging sometimes on the http getting libs